### ISSUE TITLE
translateSchema unexported name

### ISSUE THREAD

**Issue by MarErm27** [uadmin/uadmin#58]
Hi!
May I ask, why function translateSchema is unexported? What I have to do if I want to translate the schema of a model to my Html file?

About translateSchema function:
At path "\static\i18n\models" I have file auction.ru.json, which is a file for translation to the Russian language. Function translateSchema translates the model's field name to another language. My model's name is Auctions and in this file, I have a string that translates it to Аукциона
"display_name": "Translate me ---> Аукцион",
And depends on the language I want to show translated schema fields in my Html template.

**Comment by MarErm27** [comment]
So I'm trying to do this:
`uadmin.translateSchema(&auctions[0], "ru")`
But I can't because translateSchema is unexported name

**Comment by alphakitchown** [comment]
Pretty sure uadmin.CustomTranslation will work but I dont understand the point of it. From the documentation, it will create a new json file for custom translation. I will assign @dxlvgcw for the second opinion.

**Comment by MarErm27** [comment]
> 
> 
> Pretty sure uadmin.CustomTranslation will work but I dont understand the point of it. From the documentation, it will create a new json file for custom translation. I will assign @dxlvgcw for the second opinion.

Here is a error
![image](https://user-images.githubusercontent.com/56219126/78981496-b42d5700-7b28-11ea-90c2-5f281f80eb5a.png)

I'm trying to make a multilingual web application. I want to use UAdmin functions to do it. Here is the main page of my site: https://trade.3park.online/
I want it to change the language depends on the user choice.

**Comment by MarErm27** [comment]
I'm sorry I just took a look a documentation. I don't need to make a new custom translation, instead I need to use already existing translation depends on the user language

**Comment by dxlvgcw** [comment]
@MarErm27 Hi! First of all, translateSchema is private so we cannot call it outside. Public functions usually start with a capital letter like TranslateSchema. I have attempted this before then failed unfortunately.
@alphakitchown Why do you think CustomTranslation will somehow work to translate things when it only adds a new file?

**Comment by alphakitchown** [comment]
I believe uadmin.CustomTranslation will create a connection between uadmin system and the html. I have no idea how I can generate json inside the custom file.

uadmin.Translate is not a good approach because it stores inside the model.

**Comment by dxlvgcw** [comment]
Fair enough. Try to make your plan work. If you succeed, can you show us the valid proof of how CustomTranslation will somehow help to translate things? How will it differ from the json file manually created by the user?

**Comment by alphakitchown** [comment]
i have strong intuition that uadmin.CustomTranslation has something to do with it.. but it does not show the point like maybe there are more steps to do after that or i dunno?

**Comment by dxlvgcw** [comment]
I checked the source code. @MarErm27 's strategy will likely solve the problem if translate schema is public.

**Comment by alphakitchown** [comment]
alright, can you replace translateSchema to TranslateSchema for a moment to test it?

**Comment by dxlvgcw** [comment]
I am willing to do it but I do not have an app for that unfortunately.

**Comment by madong17** [comment]
Replacing function name through source code won't work on the latest version of Go because the package no longer depend on your installed package.

**Comment by alphakitchown** [comment]
alright. the best way we can do right now is to wait. let our voices be heard!!

**Comment by MarErm27** [comment]
I made a Fork for UAdmin
`https://github.com/MarErm27/uadmin`
I made a Fork for my project that will use my fork for UAdmin
`https://github.com/MarErm27/BiddingSimulatorFork`
I'm writing articles that tells what I do to make this project, if anyone interested here is a [link](https://medium.com/@ermmarat/the-web-go-golang-application-that-is-written-with-uadmin-web-framework-note-1-980597eeda90)

In my UAdmin fork I replaced translateSchema with TranslateSchema in translation.go file.

In the fork of my project I do:
go mod init github.com/MarErm27/BiddingSimulatorFork
and this:
go get -u github.com/MarErm27/uadmin@master
But I keep getting an error:
```
go: github.com/MarErm27/uadmin master => v0.4.1-0.20200410104416-4e0790d635e9
go get: github.com/MarErm27/uadmin@v0.4.1-0.20200410104416-4e0790d635e9: parsing go.mod:
        module declares its path as: github.com/uadmin/uadmin
                but was required as: github.com/MarErm27/uadmin
```

In short, I've made forks to be able to test if we can use TranslateSchema as an exported function, but I need help to resolve the issue with module declaration.

**Comment by alphakitchown** [comment]
wow i just realized that fork could be a solution, thank you.
alright so maybe comment out the go modules

**Comment by MarErm27** [comment]
![image](https://user-images.githubusercontent.com/56219126/78993264-016bf180-7b46-11ea-8f98-a61711530010.png)
It works!
The display name is a translated name of a model!
I will tell later if I will figure out how to put my fork to a module. Thank you

**Comment by MarErm27** [comment]
I made a pull request to ask to change translateSchema to TranslateSchema 
https://github.com/uadmin/uadmin/pull/59

**Comment by alphakitchown** [comment]
nice! i dont have write access to pull request so let the uadmin developers approve for you.

**Comment by uadmin** [comment]
Merged 65893df. Thanks @MarErm27 for helping.
